<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdOptions</class>
 <widget class="QDialog" name="PdOptions">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Program Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>289</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblOptionHdg">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>Set the required options for program operation.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbFocusEmpty">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;1. Set focus on Comment field if blank.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDelConf">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;2. Confirm image deletions.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDeselConf">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;3. Confirm image deselections.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAutoMove">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;4. Move to next image on Save action.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbQuit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
        <property name="shortcut">
         <string>Esc</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Save</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbFocusEmpty</tabstop>
  <tabstop>cbDelConf</tabstop>
  <tabstop>cbDeselConf</tabstop>
  <tabstop>cbAutoMove</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbQuit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbQuit</sender>
   <signal>clicked()</signal>
   <receiver>PdOptions</receiver>
   <slot>doQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSave</sender>
   <signal>clicked()</signal>
   <receiver>PdOptions</receiver>
   <slot>doSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doQuit()</slot>
  <slot>doSave()</slot>
 </slots>
</ui>
